catalog/transformer/catalog-transformer-streaming-impl/src/test/java/org/codice/ddf/transformer/xml/streaming/impl/TestXmlInputTransformer.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.transformer.xml.streaming.impl;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.codice.ddf.transformer.xml.streaming.SaxEventHandler;
+import org.codice.ddf.transformer.xml.streaming.SaxEventHandlerFactory;
+import org.codice.ddf.transformer.xml.streaming.lib.SaxEventHandlerDelegate;
+import org.junit.Test;
+import org.xml.sax.Attributes;
+import org.xml.sax.ErrorHandler;
+import org.xml.sax.SAXException;
+
+import com.vividsolutions.jts.geom.GeometryFactory;
+import com.vividsolutions.jts.io.gml2.GMLHandler;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.transform.CatalogTransformerException;
+
+public class TestXmlInputTransformer {
+    static SaxEventHandler saxEventHandler;
+    static SaxEventHandlerFactory saxEventHandlerFactory;
+
+    static GmlHandler gmlHandler;
+
+    //    XmlInputTransformer xmlInputTransformer;
+    static SaxEventHandlerDelegate saxEventHandlerDelegate;
+
+    static InputStream inputStream;
+
+    /*
+        Tests a base XmlInputTransformer, CONTENT_TYPE is null because it is not in the base xmlToMetacard mapping
+     */
+    @Test
+    public void testNormalTransform() throws FileNotFoundException, CatalogTransformerException {
+
+        inputStream = new FileInputStream(
+                "../catalog-transformer-xml/src/test/resources/metacard2.xml");
```

Comment:
This seems brittle to get a resource from another project.  If you don't want to copy that test resource, you can make that other project create a test-jar and get that resource that way.